TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/ControllerListener.java

```diff
@@ -3,7 +3,7 @@
 import com.novoda.notils.logger.simple.Log;
 import com.novoda.tpbot.Direction;
 
-interface ControllerListener {
```

Comment:
If this is being used by both I would move it to the more generic package. I know that it will move again in the future but it shows our intent clearly when looking at the structure of the app.